rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -58,6 +58,10 @@ def accounting_include_crypto2crypto():
 def accounting_taxfree_after_period():
     return YEAR_IN_SECONDS
 
+@pytest.fixture
```

Comment:
needs an extra newline here according to `flake8`. Fails the linting test [in travis](https://travis-ci.org/rotkehlchenio/rotkehlchen/jobs/438377164#L845)